File: PB - More Exercises Python/7.0 Nested Loops - More Exercises/11. HappyCat Parking.py
days, hours = [int(input()) + 1 for _ in range(2)]
current_tax, total_sum = 0, 0

for day in range(1, days):
    for hour in range(1, hours):
        if not day & 1 and hour & 1:
            current_tax += 2.50
        elif day & 1 and not hour & 1:
            current_tax += 1.25
        else:
            current_tax += 1.00
    print(f'Day: {day} - {current_tax:.2f} leva')
    total_sum += current_tax
    current_tax = 0

print(f'Total: {total_sum:.2f} leva')


'''
2
5
'''

File: PB - More Exercises GO/7.0 Nested Loops - More Exercises/11. HappyCat Parking.go
package main

import "fmt"

func main() {
	var days, hours, currentTax, totalSum float64
	fmt.Scan(&days, &hours)

	for day := 1; day <= int(days); day++ {
		for hour := 1; hour <= int(hours); hour++ {
			if day%2 == 0 && hour%2 != 0 {
				currentTax += 2.50
			} else if day%2 != 0 && hour%2 == 0 {
				currentTax += 1.25
			} else {
				currentTax += 1.00
			}
		}
		fmt.Printf("Day: %v - %.2f leva\n", day, currentTax)
		totalSum += currentTax
		currentTax = 0
	}

	fmt.Printf("Total: %.2f leva\n", totalSum)
}

/*
name   : pacman -Qi qt5-base qt5ct
input  :2 5
output :
Day: 1 - 5.50 leva
Day: 2 - 9.50 leva
Total: 15.00 leva
*/
